test/unit/content-src/components/DiscoveryStreamBase.test.jsx

```diff
@@ -87,18 +87,6 @@ describe("<DiscoveryStreamBase>", () => {
     globals.restore();
   });
 
-  it("should render null if spocs not loaded", () => {
-    wrapper = mountComponent({spocs: {loaded: false, data: {spocs: null}}});
-
-    assert.equal(wrapper.type(), null);
-  });
-
-  it("should render null if feeds not loaded", () => {
-    wrapper = mountComponent({feeds: {loaded: false}});
-
-    assert.equal(wrapper.type(), null);
-  });
-
```

Comment:
Does checking for notNull as we continuing minimal rendering even when spocs and feeds are not loaded help?